pg_search/src/postgres/insert.rs

```diff
@@ -69,6 +76,9 @@ pub unsafe fn init_insert_state(
     index_info: *mut pg_sys::IndexInfo,
     writer_resources: WriterResources,
 ) -> *mut InsertState {
+    if IsLogicalWorker() {
```

Comment:
We talked about putting this in `ambuild` instead, right

Reply:
I thought @eeeebbbbrrrr suggested during initialization of `InsertState`?

Reply:
yeah, it has to be in InsertState b/c that's called for INSERT/UPDATE statements.

Logical Replication would never call `ambuild()`